.github/workflows/php.yml

```
@@ -1,55 +1,52 @@
+---
 name: Unit tests
 
 on:
   push:
+    paths:
+      - '**.php'
+      - 'composer.json'
     branches:
-      - 'master'
-    tags:
       - '*'
+      - '!v1'
+      - '!v2'
```

I'd prefer keeping the `tags` configuration, here.

```suggestion
      - '!v2'
    tags:
      - '*'
```

---

That doesn't make much sense Imho.

---

Care to elaborate?

---

There's just absolutely no need to perform another test upon tag creation. Creating a tag requires a push or a merged PR and in both cases, the tests have already been performed.